recipes/cmake/3.x.x/conanfile.py

```diff
@@ -54,6 +74,9 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "doc"))
 
+    def package_id(self):
+        self.info.options.with_openssl = self._with_openssl
```

Comment:
Good point. I remember that we have other packages using "auto", we should update it in a next opportunity.